README.md

```diff
@@ -127,10 +131,31 @@ Please install the ```tree``` command before moving forward:
         ```
 
    - MacOSX
-        ```
-        install tree - http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
-        install wget - http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
-        ```
+       
+        - Option 1:
+        
+          - install tree
+          
+          http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
+        
+          - install wget 
+          
+          http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
+       
+        - Option 2 (Via brew ):
```

Comment:
Do you want to throw light on the option 2, you mentioned you found you needed to do this on a version(s) of Mac OS X